bundle/DependencyInjection/Compiler/WebpFormatVariationPathGeneratorDecoratorPass.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\DependencyInjection\Compiler;
+
+use Netgen\Bundle\SiteBundle\Core\Imagine\VariationPathGenerator\WebpFormatVariationPathGenerator;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+class WebpFormatVariationPathGeneratorDecoratorPass implements CompilerPassInterface
+{
+    /**
+     * Overrides the IO resolver to disable generating absolute URIs to images.
+     */
+    public function process(ContainerBuilder $container): void
+    {
+        if ($container->has('ezpublish.image_alias.variation_path_generator')) {
+
```

Comment:
Could you please remove the extra empty line?

Reply:
This should use an early return:

```
if (!$container->has(...)) {
    return;
}
```

Reply:
Fixed in 99a67ec65370ef05cc0d411fb31d1baf00cb119f